infrastructure/resource_group_level/modules/trade-finance-manager-api.bicep

```diff
@@ -0,0 +1,247 @@
+param location string
+param environment string
+param containerRegistryName string
+param appServicePlanEgressSubnetId string
+param appServicePlanId string
+param privateEndpointsSubnetId string
+param cosmosDbAccountName string
+param cosmosDbDatabaseName string
+param logAnalyticsWorkspaceId string
+param externalApiHostname string
+param dtfsCentralApiHostname string
+param storageAccountName string
+
+var dockerImageName = '${containerRegistryName}.azurecr.io/trade-finance-manager-api:${environment}'
+var dockerRegistryServerUsername = 'tfs${environment}'
+
+// These values are taken from GitHub secrets injected in the GHA Action
+@secure()
+param secureSettings object = {
+
+}
+
+// These values are taken from an export of Configuration on Dev (& validating with staging).
+@secure()
+param additionalSecureSettings object = {
+  DOCKER_REGISTRY_SERVER_PASSWORD: 'test-value'
+  UKEF_INTERNAL_NOTIFICATION: 'test-value'
+  // TODO:FN-424 All of these were found on dev but not staging, remove if appropriate
+  DTFS_CENTRAL_API_KEY: 'test-value'
+  EXTERNAL_API_KEY: 'test-value'
+  JWT_VALIDATING_KEY: 'test-value'
+  PORTAL_API_KEY: 'test-value'
+  TFM_API_KEY: 'test-value'
+}
+
+// These values are taken from GitHub secrets injected in the GHA Action
+@secure()
+param secureConnectionStrings object = {
+  // NOTE that CORS_ORIGIN is not present in the variables exported from dev or staging
+  CORS_ORIGIN: 'test-value'
+  UKEF_TFM_API_SYSTEM_KEY: 'test-value'
+  UKEF_TFM_API_REPORTS_KEY: 'test-value'
+  // TODO:FN-429 Note that this has a value like https://tfs-dev-tfm-fd.azurefd.net 
+  // while in the CLI it is injected as a secret, we can probably calculate it from the Front Door component.
+  TFM_URI: 'test-value'
```

Comment:
Again, I've put a TODO as I don't think this should be a secret and should just be generated from the Front-Door deployment. (`TFM_URI` isn't used as an input to the Front-Door component.)